### PR TITLE
Create Plutonication.md

### PR DESCRIPTION
**Pull request by RostislavLitovkin**
#### Project Abstract

- Communications protocol that enables seamless interactions between dApps and wallets across all platforms.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [x] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@rostislavlitovkin:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by RostislavLitovkin** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for the application. I have a couple of initial questions/comments:
- We usually don't pay for hosting costs. Could you remove this? 
- What do you mean by Mockups? 
- Are you aware of [SubConnect](https://github.com/Koniverse/SubConnect), [Tesseract](https://github.com/tesseract-one/Tesseract.rs) or [@talismn/connect](https://github.com/TalismanSociety/talisman-connect)?

File: applications/Plutonication.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+### Contact
+
+- **Contact Name:** Rostislav Litovkin
+- **Contact Email:** rostislavlitovkin@gmail.com
+- **Website:** https://github.com/cisar2218/Plutonication
+
+### Legal Structure
+
+- **Registered Address:** None
+- **Registered Legal Entity:** None
```
Comment (semuelle) [inline applications/Plutonication.md:216]: Please provide at least an address. If you prefer not to do so publicly, feel free to [email us](mailto:grants@web3.foundation).

**Review by semuelle — CHANGES_REQUESTED** [review]

**Review by semuelle — CHANGES_REQUESTED** [review]
Please see my questions in the chat.

**Review by RostislavLitovkin — COMMENTED** [review]
Added all missing info

File: applications/Plutonication.md [hunk 2]
```diff
@@ -0,0 +1,482 @@
+# Plutonication
+
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines (except for the title)! Lines starting with a `>` (such as this one) should be removed. Please use markdown instead of HTML (e.g. `![](image.png)` instead of `<img>`).
```
Comment (keeganquigley) [inline applications/Plutonication.md:3]: As the text states, this part can be removed. Feel free to take it out.

File: applications/Plutonication.md [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+- **Website:** https://github.com/cisar2218/Plutonication
+
+### Legal Structure
+
+- **Registered Address:** Píškova 1946/12, 155 00, Prague, Czech republic
+- **Registered Legal Entity:** None
+
+### Team's experience
+
+Please describe the team's relevant experience. If your project involves development work, we would appreciate it if you singled out a few interesting projects or contributions made by team members in the past.
```
Comment (keeganquigley) [inline applications/Plutonication.md:237]: Could you remove this template text as well?

**Comment by keeganquigley** [comment]
Thanks for the application @RostislavLitovkin in addition to my above comments:

- Wallet Connect does have both a [Dapp integration guide](https://docs.walletconnect.com/advanced/multichain/polkadot/dapp-integration-guide) and a [wallet integration guide](https://docs.walletconnect.com/advanced/multichain/polkadot/wallet-integration-guide) for Polkadot. Can you compare to this and what is missing that your solution would provide?
- You point out that the only way to connect a mobile wallet to a Dapp is through Substrate Connect or Vault, but afaik mobile wallets such as Talisman and Fearless will also let you connect.
- How do you plan to get developers to use your solution?

**Review by semuelle — APPROVED** [review]

**Review by keeganquigley — APPROVED** [review]
Left comment in private channel.

**Review by semuelle — APPROVED** [review]

**Review by nikw3f — APPROVED** [review]
Happy to support

**Pull request merged**